src/components/Proposal/ProposalHistoryRow.tsx

```diff
@@ -174,31 +176,27 @@ class ProposalHistoryRow extends React.Component<IProps, IState> {
           />
         </td>
         <td onClick={this.gotoProposal} className={closeReasonClass}>
-          <div className={css.decisionPassed}>
-            <img src="/assets/images/Icon/vote/for.svg"/>
-            <span>Passed</span>
-            <div className={css.decisionReason}>
-              <span>{closeReason}</span>
-            </div>
-          </div>
-          <div className={css.decisionFailed}>
-            <img src="/assets/images/Icon/vote/against.svg"/>
-            <span>Failed</span>
-            <div className={css.decisionReason}>
-              <span>{closeReason}</span>
-            </div>
-          </div>
+          {(passed || failed) ?
+            <div className={passed ? css.decisionPassed : css.decisionFailed}>
+              <img src={`/assets/images/Icon/vote/${passed ? "for.svg" : "against.svg"}`} />
+              <span>{passed ? "Passed" : "Failed"}</span>
+              <div className={css.decisionReason}>
+                <span>{closeReason}</span>
+              </div>
+            </div> :
+            <div className={css.decisionInProgress}>In Progress</div>
```

Comment:
how bout change that to the actual phase ? 
que, preboosted, boosted

Reply:
Possible reason:  1) It was not requested by the ticket, and 2) it adds risk which we're trying to minimize in Alchemy 1.0 releases.  Could be done for 2.0.

Reply:
Could be done for 2.0 except that it makes no sense since we haven't yet worked https://github.com/daostack/alchemy/pull/2104 (and therefore active proposals don't show up there in the history page)

Reply:
Please provide guidance:  @orenyodfat

Reply:
lets do que, preboosted, boosted

Reply:
Done